kie-server-parent/kie-server-router/kie-server-router-proxy/src/main/java/org/kie/server/router/KieServerRouter.java

```diff
@@ -107,10 +126,42 @@
     public KieServerRouter() {
         configRepositoryServiceLoader.forEach(repo -> repository = repo);
         log.info("KIE Server router repository implementation is " + repository);
+        MANAGEMENT_SECURED = Boolean.parseBoolean(System.getProperty(KIE_ROUTER_MANAGEMENT_SECURED, "false"));
+        IDENTITY_PROVIDER = System.getProperty(KieServerRouterConstants.KIE_ROUTER_IDENTITY_PROVIDER, "default");
     }
 
-    public static void main(String[] args) {
+    public static void main(String[] args) throws Exception {
+        Options options = new Options();
+
+        Option addInstanceOption = Option.builder("addInstance")
```

Comment:
Would it be possible to separate these user adding/deleting into separate class with main method?
That would help functionality separation as this main method already takes care of starting Smart router.

Reply:
sure

Reply:
@sutaakar  I don't see this clear now... it is not the same to do java -jar <jar name> options than java -jar <jar name> <class> <options> ... I think it is more cumbersome

Reply:
@elguardian Do you plan some changes related to options?

Reply:
@sutaakar as I stated before. No, it does not make sense; it ise less user friendly.